projects/hello_world/project.mk

```diff
@@ -1,3 +1 @@
-USER_TESTS += $(LIBRARY_DIR)/L1_Peripheral/cortex/test/dwt_counter_test.cpp
-USER_TESTS += $(LIBRARY_DIR)/L1_Peripheral/cortex/test/interrupt_test.cpp
-USER_TESTS += $(LIBRARY_DIR)/L1_Peripheral/test/interrupt_test.cpp
+USER_TESTS += $(LIBRARY_DIR)/L2_HAL/sensors/battery/test/ltc4150_test.cpp
```

Comment:
I think you did this for testing. Can you revert to previous tests

Reply:
Done. Sorry, i forgot to revert it